include/ttmlir/Scheduler/Scheduler.h

```diff
@@ -23,6 +23,9 @@ class Scheduler {
   // Method to get the next set of schedulable operations
   llvm::SmallVector<mlir::Operation *> getScheduleableOps();
 
+  // Method to check if an operation is a TT schedulable
+  bool isTTShedulableOp(mlir::Operation *op);
```

Comment:
Method to check if an operation is a TTIR or TTNN op

Reply:
Can you explain further what do you mean by this comment? The idea here was to expose the `isTTShedulableOp(mlir::Operation *op)` as a public method and, therefore, in the `L1InrterleavedPolicy` discard non-analyzable ops (e.g. `EmptyOp`s).

Reply:
Sorry, i wasn't clear here, it was just a nitpick. I meant that "Method to check if an operation is a TTIR or TTNN op" might be a better comment for the method as it provides more details on what the method does. With the current name and comment i had to got to the implementation to actually understand what it checks.